### PR TITLE
Fix: Remove duplicate French and Slovak town names

### PR DESCRIPTION
**Pull request by nikolas**
I've replaced some of the duplicate real town names with new real town
names in the French and Slovak sets.

Also, some Slovak town names were missing accents, so I've fixed those.

### CODE REVIEW

File: src/table/townname.h [hunk 1]
```diff
@@ -2060,8 +2060,8 @@ static const char * const _name_slovak_real[] = {
 	"Humenne",
 	"Hurbanovo",
 	"Kezmarok",
-	"Komarno",
-	"Kosice",
+	"Kom\xC3\xA1rno",
+	"KoxC5\xA1ice",
```
Comment (LordAro) [inline src/table/townname.h]: this one looks like it's missing a \

**Review by LordAro — DISMISSED** [review]

File: src/table/townname.h (hunk 1 above)
Comment (nikolas) [reply src/table/townname.h]: woops thanks - fixed.

**Review by planetmaker — APPROVED** [review]
I can't exactly judge the correctness of the accents in Romanian... but yes!

**Comment by nikolas** [comment]
I'm not the most familiar with these town names either - the Slovak corrections are based on info found here: https://en.wikipedia.org/wiki/List_of_cities_and_towns_in_Slovakia

**Pull request merged**